### PR TITLE
Update RPYC tutorial based on @ndward's input

### PR DESCRIPTION
**Pull request by ddemidov**

### CODE REVIEW

**Comment by ddemidov** [comment]
Preview: http://ev3dev-lang.readthedocs.io/projects/python-ev3dev/en/update-rpyc-tutorial/rpyc.html

**Comment by ddemidov** [comment]
Pinging @ndward

File: docs/rpyc.rst [hunk 1]
```diff
@@ -9,14 +9,24 @@ processes and computers, so that remote objects can be manipulated as if they
...
-   command (depending on your operating system, you may need to use ``pip3`` or
-   ``pip`` on the PC instead of ``easy_install3``):
+1. Install RPyC both on the EV3 and on your desktop PC. On the EV3 the following
+   command should work:
 
    .. code-block:: shell
 
        sudo easy_install3 rpyc
 
+   On the desktop PC, it really depends on your operating system. In case its
```
Comment (WasabiFan) [inline docs/rpyc.rst]: "its" should be "it's" (here and below)

File: docs/rpyc.rst [hunk 2]
```diff
@@ -30,9 +40,18 @@ use RPyC with ev3dev:
 
       chmod +x rpyc_server.sh
 
-   Launch the created file either from ssh session, or from brickman.
+   Launch the created file either from ssh session (with ``./rpyc_server.sh``
```
Comment (WasabiFan) [inline docs/rpyc.rst]: Can you link to the SSH tutorial here for those who haven't already configured it?

File: docs/rpyc.rst (hunk 1 above)
Comment (ddemidov) [inline docs/rpyc.rst]: I think here 'its' is a short form of 'it is', so no apostrophe? But I guess I should change it to 'it is' anyway.

File: docs/rpyc.rst (hunk 2 above)
Comment (ddemidov) [inline docs/rpyc.rst]: Linked, not here, but in the 'installation' step.

File: docs/rpyc.rst (hunk 1 above)
Comment (ddemidov) [inline docs/rpyc.rst]: I think [wrong](http://www.elearnenglishlanguage.com/blog/english-mistakes/its/) :smiley_cat:

Comment (WasabiFan) [inline docs/rpyc.rst]: Yeah, English is weird :laughing: Anything with an apostrophe is a contraction, so "it's" is the version that expands into two words. "its" is the possessive, used like "his."

**Pull request merged**